proto/ibc/core/client/v1/tx.proto

```diff
@@ -82,16 +82,17 @@ message MsgUpgradeClientResponse {}
 
 // MsgSubmitMisbehaviour defines an sdk.Msg type that submits Evidence for
 // light client misbehaviour.
+// Warning: DEPRECATED
```

Comment:
Should we add something like: "Use MsgUpdateClient instead, this will be removed in a future release"